src/nyx/featureset.h

```diff
@@ -450,7 +450,7 @@ class FeatureSet
 			m_enabledFeatures[STAT_FERET_DIAM_MODE] = true;
 	}
 	bool isEnabled(int fc) const { return fc < AvailableFeatures::_COUNT_ ? m_enabledFeatures[fc] : false; }
-	bool anyEnabled(std::initializer_list<int> F) const
+	bool anyEnabled(const std::initializer_list<AvailableFeatures> F) const

```

Comment:
should we pass this by reference?

Reply:
It's not a typo, just a habit from the early times of std::initializer_list when passing a constant list by reference caused a CL's hiccup. Fixed